stats/view.go

```diff
@@ -26,8 +26,8 @@ import (
 	"go.opencensus.io/tag"
 )
 
-// View allows users to filter and aggregate the recorded events
-// over a time window. Each view has to be registered to enable
+// View allows users to filter and aggregate the recorded events.
+// Each view has to be registered to enable
 // data retrieval. Use NewView to initiate new views.
```

Comment:
Perhaps we can backspace and move the `data retrieval` line to the end of the previous line.